app/src/main/res/values/strings.xml

```diff
@@ -595,4 +596,59 @@ Signature: %5$s</string>
+    <string name="summary">Summary</string>
+
+    <!-- Event Activity -->
+    <string name="show_booked_events">Show booked events</string>
+    <string name="events_tickets">Events &amp; Tickets</string>
+    <string name="ticket_type">Ticket Type</string>
+    <string name="price">Price</string>
+    <string name="location">Location</string>
+    <string name="event" translatable="false">Event</string>
+    <string name="payment_selection">Payment Selection</string>
+    <string name="add_eventcalendar">Add In Calendar</string>
+    <string name="all_events">All Events</string>
+    <string name="booked_events">Booked Events</string>
+    <string name="link">VISIT WEBSITE</string>
+    <string name="tum_calendar">TUM Calendar</string>
+    <string name="external_calendar">External Calendar</string>
+    <string name="add_to_calendar_info">To which calendar do you want to add the event?</string>
+    <string name="event_export" translatable="false">Event Export</string>
+
+    <!-- Payment -->
+    <string name="creditcard">Credit Card</string>
+    <string name="complete_purchase">Buy now</string>
+    <string name="saved_cards">Please select a payment method</string>
+    <string name="enter_card_info_title">Payment Information</string>
+    <string name="enter_card_info_description">Please enter your payment information to conclude the purchase:</string>
+    <string name="purchase_progress_message">Requesting Ticket, please wait…</string>
+    <string name="purchase_success_title">Purchase successful</string>
+    <string name="purchase_success_message">You can access your ticket in the ticket section.</string>
+    <string name="purchase_success_continue">Continue</string>
+    <string name="purchase_error_message">Failure during server call.</string>
+    <string name="card_data_invalid">The card data is invalid.</string>
+    <string name="customersession_init_failed">Failed to load saved credit cards.</string>
+    <string name="cardholer">Cardholder:</string>
+    <string name="payment_info_price_reminder">Pay {0} with:</string>
+    <string name="internal_error">Internal Failure</string>
+    <string name="creditcard_ending_in">ending in</string>
+    <string name="empty_cardholder_message">No cardholder specified.</string>
+    <string name="sorry">Sorry</string>
+    <string name="not_logged_in_message">You have to be logged in and provide access to your name and TUM-ID (via TUM-Online token) to purchase tickets.</string>
+    <string name="ticket_retrieval_error">Failure while receiving ticket from server.</string>
+    <string name="reserving_ticket">Reserving your ticket, please wait...</string>
+    <string name="tickets_left" formatted="true">%d Tickets left!</string>
+    <string name="ticket_contingent_exhausted">All tickets of the requested type are already sold.</string>
+
+    <string name="redeemed">Redeemed</string>
+    <string name="no_network_connection">Currently no connection to server possible</string>
```

Comment:
Can you please use the already existing `no_internet_connection` string?

Reply:
right, changed that in #989